lvk/vulkan/VulkanClasses.cpp

```diff
@@ -1410,7 +1417,10 @@ lvk::Result lvk::VulkanSwapchain::present(VkSemaphore waitSemaphore) {
       .pSwapchains = &swapchain_,
       .pImageIndices = &currentImageIndex_,
   };
-  VK_ASSERT_RETURN(vkQueuePresentKHR(graphicsQueue_, &pi));
+  VkResult r = vkQueuePresentKHR(graphicsQueue_, &pi);
+  if (r != VK_SUCCESS && r != VK_SUBOPTIMAL_KHR) {
```

Comment:
Is it still happening when we do not call `swapchain_->getCurrentTexture()` in the beginning?

Reply:
yes